blivet/udev.py

```diff
@@ -28,19 +28,25 @@
 from . import util
 from .size import Size
 from .flags import flags
+from .nvdimm import nvdimm
 
 import pyudev
 global_udev = pyudev.Context()
 
 import gi
 gi.require_version("BlockDev", "2.0")
+gi.require_version("GLib", "2.0")
 from gi.repository import BlockDev as blockdev
+from gi.repository import GLib
 
 import logging
 log = logging.getLogger("blivet")
 
-INSTALLER_BLACKLIST = (r'^mtd', r'^mmcblk.+boot', r'^mmcblk.+rpmb', r'^zram', r'ndblk')
+INSTALLER_BLACKLIST = [r'^mtd', r'^mmcblk.+boot', r'^mmcblk.+rpmb', r'^zram', r'ndblk']
 """ device name regexes to ignore when flags.installer_mode is True """
+# we don't have NVDIMM plugin, just ignore all pmem devices too
+if not nvdimm.nvdimm_plugin_available:
+    INSTALLER_BLACKLIST.append(r'pmem')
```

Comment:
Will it be safer if we use `r'^pmem'` instead? (I think we should have done the same for 'ndblk' above as well, but we didn't.)

Reply:
Probably, the nvdimm devices names should look like `pmemX` or `pmemXs`. We originally had only `pmem` there (https://github.com/storaged-project/blivet/pull/699/files#diff-48651c0ada40d943223dea89af3e129aL38). I'm not sure if there was a reason for that.

Reply:
I think it was just an oversight. It should have been `r'^pmem'`.

Reply:
Ok, fixed.